scripts/create-example-pages

```diff
@@ -7,7 +7,6 @@
 
 const fs = require('fs');
 const examples = require('../site/_data/examples.json');
-const svg2png = require('svg2png-many');
```

Comment:
Why changing these?

Reply:
Yeah, somewhat unrelated but I noticed we are compiling pngs anyway so we don't need the extra library anymore.